aries-site/src/examples/components/layer/LayerExample.js

```diff
@@ -9,33 +9,50 @@ import {
   useConfirmation,
 } from './components';
 
-export const LayerExample = () => (
-  <ConfirmationProvider>
-    <ConfirmationContext.Consumer>
-      {({ showLayer, showConfirmation, setShowLayer }) => (
-        <>
-          <Box align="center">
-            <Button
-              label="Show me the layer"
-              onClick={() => setShowLayer(true)}
-              primary
-            />
-          </Box>
-          {showLayer ? <AddApplication /> : null}
-          {showConfirmation ? <DoubleConfirmation title="application" /> : null}
-        </>
-      )}
-    </ConfirmationContext.Consumer>
-  </ConfirmationProvider>
-);
+export const LayerExample = () => {
+  const announce = useContext(AnnounceContext);
+
+  return (
+    <ConfirmationProvider>
+      <ConfirmationContext.Consumer>
+        {({ showLayer, showConfirmation, setShowLayer }) => {
+          return (
+            <>
+              <Box align="center">
+                <Button
+                  label="Show me the layer"
+                  onClick={() => {
+                    announce('Add application modal opened.', 'polite');
+                    setShowLayer(true);
+                  }}
+                  primary
+                />
+              </Box>
+              {showLayer ? <AddApplication /> : null}
+              {showConfirmation ? (
+                <DoubleConfirmation title="application" />
+              ) : null}
+            </>
+          );
+        }}
+      </ConfirmationContext.Consumer>
+    </ConfirmationProvider>
+  );
+};
 
 const AddApplication = ({ ...rest }) => {
+  const announce = useContext(AnnounceContext);
   const { onClose } = useConfirmation();
 
+  const closeLayer = () => {
+    announce('Add application modal closed.', 'polite');
```

Comment:
Only thing I noticed was that this announce was difficult to perceive, because the button's label would immediately get read.

The "modal opened" reads great.

What's your thought on "aggressive" vs. "polite" in this scenario?

Reply:
Yes tested both 'polite' and 'assertive' here, but in both cases the label's announcement takes precedence. See comment in PR description

> The close layer announcement gets interrupted by the button label announcement, no great way around this unless we delayed moving focus to the button after the layer closes which isn't ideal. So I left as is, I think the more important announcement is when the layer opens.